README.md

```diff
@@ -1,3 +1,9 @@
 # angular-revealable
 
 Angular directive to make revealing password input easy
+
+## Use
+
+`<input type="password" data-revealable>`
+
+This will create an icon on the right of the input field that when selected shows the current password value below the input field. There is some basic classes applied for styling purposes. The icon used depends on fontawesome.
```

Comment:
I guess you can't draw an eye very easily, but I still hate the idea of including an entire font library for one icon.